mne/viz/backends/_pyvista.py

```diff
@@ -243,6 +245,35 @@ def subplot(self, x, y):
     def scene(self):
         return self.figure
 
+    def _orient_lights(self):
+        lights = list(self.plotter.renderer.GetLights())
+        lights.pop(0)  # unused headlight
+        lights[0].SetPosition(_to_pos(45.0, -45.0))
+        lights[1].SetPosition(_to_pos(-30.0, 60.0))
+        lights[2].SetPosition(_to_pos(-30.0, -60.0))
+
+    def update_lighting(self):
+        # Inspired from Mayavi's version of Raymond Maple 3-lights illumination
+        lights = list(self.plotter.renderer.GetLights())
+        headlight = lights.pop(0)
```

Comment:
I'm a bit concerned that there are two `.pop`'s going on. Is it worth adding the conditional:
```
assert len(lights) in (3, 4)
if len(lights) == 4:
    lights.pop(0)
```
both places?

Reply:
ohhh wait you don't actually remove the light from VTK. Never mind!